test/test_utils.py

```diff
@@ -579,6 +580,22 @@ def test_limit_length(self):
             limit_length('foo bar baz asd', 12).startswith('foo bar'))
         self.assertTrue('...' in limit_length('foo bar baz asd', 12))
 
+    def test_parse_codecs(self):
+        self.assertEqual(parse_codecs(''), {})
+        self.assertEqual(parse_codecs('avc1.77.30, mp4a.40.2'), {
+            'vcodec': 'avc1.77.30',
+            'acodec': 'mp4a.40.2',
+        })
+        self.assertEqual(parse_codecs('mp4a.40.2'), {
+            'vcodec': 'none',
+            'acodec': 'mp4a.40.2',
+        })
+        # http://once.unicornmedia.com/now/master/playlist/bb0b18ba-64f5-4b1b-a29f-0ac252f06b68/77a785f3-5188-4806-b788-0893a61634ed/93677179-2d99-4ef4-9e17-fe70d49abfbf/content.m3u8
+        # self.assertEqual(parse_codecs('mp4a.40.5,avc1.42001e'), {
```

Comment:
this are some codecs that i know about:
vcodecs: avc1, vp9, vp8, avc3, hev1, h263, h264, mp4v
acodecs: mp4a, opus, vorbis, mp3, aac
is there other codecs to add?